core/internal/runhistory/runhistory.go

```diff
@@ -121,9 +115,27 @@ func (rh *RunHistory) Flatten() ([]*service.HistoryItem, error) {
 	return history, nil
 }
 
-// Clone returns a deep copy of the history tree.
-func (rh *RunHistory) Tree() pathtree.TreeData {
-	return rh.pathTree.Tree()
+// GetNumber returns the value of a number-valued metric.
+func (rh *RunHistory) GetNumber(path ...string) (float64, bool) {
+	value, exists := rh.pathTree.GetLeaf(pathtree.TreePath(path))
+	if !exists {
+		return 0, false
+	}
+
+	switch x := value.(type) {
+	case int64:
```

Comment:
do you need to handle any other ints?

Reply:
no---int64 and float64 are the only possibilities. With encoding/json (and segmentio) it's always float64 actually---but I'm preparing for my simplejsonext PR which can also unmarshal to int64.

Reply:
got it thanks!